puppet/modules/profiles/manifests/monitoring/client.pp

```diff
@@ -0,0 +1,43 @@
+class profiles::monitoring::client (
+  String[1] $prometheus_username,
+  String[1] $prometheus_password,
+  Stdlib::HTTPUrl $prometheus_url = 'https://prometheus-prod-01-eu-west-0.grafana.net/api/prom/push',
+  Array[Hash] $blackbox_targets = [],
+) {
+  if $facts['os']['family'] == 'RedHat' {
+    yumrepo { 'copr:copr.fedorainfracloud.org:evgeni:node-exporter-textfile-collector-scripts':
```

Comment:
Should we move this to `theforeman` so it's not tied to just you?

Reply:
I use it for my private infra too, but don't care where it lives, so yeah.

(@theforeman/infra or something?)

Reply:
Makes sense. We used to have https://yum.theforeman.org/infra/ but that was from Koji and we can delete it by now. Having a replacement on copr makes sense to me.

Reply:
Moved to https://copr.fedorainfracloud.org/coprs/g/theforeman/infra/